kagefunc.py

```diff
@@ -85,12 +85,12 @@ def generate_keyframes(clip: vs.VideoNode, out_path=None) -> None:
     # Speed up the analysis by resizing first. Converting to 8 bit also seems to improve the accuracy of wwxd.
     clip = core.resize.Bilinear(clip, 640, 360, format=vs.YUV420P8)
     clip = core.wwxd.WWXD(clip)
-    out_txt = "# WWXD log file, using qpfile format\n\n"
+    out_txt = "# WWXD log file, using qpfile format\n\n" if header else ""
     for i in range(clip.num_frames):
         if clip.get_frame(i).props.Scenechange == 1:
-            out_txt += "%d I -1\n" % i
-        if i % 1000 == 0:
-            print(i)
+            out_txt += f"{i} I -1\n" if i != 0 else ""
```

Comment:
If you just want to exclude the first frame, start the range two lines above at 1 instead of 0